.github/workflows/sycl_precommit.yml

```diff
@@ -49,13 +100,13 @@ jobs:
     uses: ./.github/workflows/sycl_gen_test_matrix.yml
     with:
       ref: ${{ github.event.pull_request.head.sha }}
-      lts_config: "ocl_x64;ocl_gen9"
+      lts_config: "lin_intel"
```

Comment:
I think we can stay the way we were before so that our test continue to run on a machine with a CPU and one with a GPU. For our branch can you not do this particular change and related ones ?

Reply:
reverted this particular change.